### PR TITLE
Add School Calendar

### PR DESCRIPTION
**Pull request by mbacvanski**
Closes issue #3

### CODE REVIEW

File: web/config/webpack.config.dev.js [hunk 1]
```diff
@@ -150,12 +150,12 @@ module.exports = {
           name: 'static/media/[name].[hash:8].[ext]'
         }
       },
-      // "url" loader works like "file" loader except that it embeds assets
+      // "calendarURL" loader works like "file" loader except that it embeds assets
```
Comment (plusCubed) [inline web/config/webpack.config.dev.js]: What are these changes in the webpack config for?

Comment (mbacvanski) [inline web/config/webpack.config.dev.js]: Pretty sure I didn't touch that .. is it important?

Comment (plusCubed) [inline web/config/webpack.config.dev.js]: We don't have any images yet, but yeah, for loading images.

Comment (mbacvanski) [inline web/config/webpack.config.dev.js]: My bad, did some sketchy refactoring on other variables with the same name. Changing it back now.

**Pull request merged**